test/e2e/autoscale.go

```diff
@@ -367,8 +373,17 @@ func checkPodScale(ctx *TestContext, targetPods, minPods, maxPods float64, done
 // The given `duration` is how long the traffic will be generated. You must make sure that the signal
 // from the given `done` channel will be sent within the `duration`.
 func AssertAutoscaleUpToNumPods(ctx *TestContext, curPods, targetPods float64, done <-chan time.Time, quick bool) {
-	ctx.t.Helper()
+	wait := AutoscaleUpToNumPods(ctx, curPods, targetPods, done, quick)
```

Comment:
Why drop `t.Helper()`?

Reply:
I don't mind dropping or keeping it. So added it back in a new commit, together with the other two occurrences.